lib/check/validate.js

```diff
@@ -127,6 +127,15 @@ function warn(ctx, reference) {
   let index = -1
 
   while (++index < reference.nodes.length) {
-    reference.file.message(reason, reference.nodes[index], origin)
+    const message = reference.file.message(
+      reason,
+      reference.nodes[index],
+      origin
+    )
+    message.url = 'https://github.com/remarkjs/remark-validate-links#readme'
+
+    if (suggestion) {
+      message.expected = [suggestion]
```

Comment:
I believe (most?) everything that has suggestions (`expected`) also has an `actual` (https://github.com/vfile/vfile-message#actual)? Should that be here too?
Otherwise, good

Reply:
We may have that info.
The starts would be link start + title length + 3.
The end would be link start + title length + 3 + url length.

Reply:
Yes, I also concluded whitespace is an issue. I initially thought this would be a very simple feature, but since there are some edge cases, I decided to revert them. Now the PR only adds the url.